app/config/services.yml

```diff
@@ -24,6 +24,11 @@ services:
         tags:
             - { name: twig.extension }
 
+    app.twig.extension_intl:
+        class: Twig_Extensions_Extension_Intl
```

Comment:
@stof Does [app.controller_listener](https://github.com/symfony/symfony-demo/blob/master/app/config/services.yml#L33-L37) should be private too?